rotkehlchen/chain/arbitrum_one/modules/arbitrum_one_bridge/accountant.py

```diff
@@ -0,0 +1,32 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.accounting.structures.evm_event import get_tx_event_type_identifier
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.arbitrum_one.constants import CPT_ARBITRUM_ONE
+from rotkehlchen.chain.evm.accounting.interfaces import ModuleAccountantInterface
+from rotkehlchen.chain.evm.accounting.structures import TxEventSettings
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.pot import AccountingPot
+
+
+class ArbitrumOneBridgeAccountant(ModuleAccountantInterface):
+
+    def event_settings(self, pot: 'AccountingPot') -> dict[str, TxEventSettings]:  # pylint: disable=unused-argument  # noqa: E501
+        """Being defined at function call time is fine since this function is called only once"""
```

Comment:
The get_tx_event_type_identifier seems to take into consideration the counterparty but since I'm not familiar with the accounting part of the code and optimism has also two identical accountants, I leave it as it is (similar to optimism)